lib/modules/manager/pep621/processors/rye.ts

```diff
@@ -0,0 +1,278 @@
+import is from '@sindresorhus/is';
```

Comment:
Please add a draft of the user-facing documentation. Follow these steps:

1. Create a `readme.md` file, like this: `lib/modules/manager/pep621/readme.md`.
2. Explain in the readme:
    - what the manager does
    - which files the manager can update
    - if/what/how the user must configure things to get it working
    - the limitations of the manager, things it can not update, or only under certain conditions
    
It's to have a rough draft for now. We can clean it up during the review later.

Reply:
I'll get to that once https://github.com/containerbase/base/pull/2684 starts working. That's currently where my time is being wasted on :)

Reply:
I'll let that other PR waste your time first then. :melting_face:

Please keep this review comment open, for now. You may resolve this comment after you added a draft of the docs.